.gitignore

```diff
@@ -0,0 +1 @@
+mc_data
```

Comment:
Why is this needed?

Reply:
Just to avoid accidental commits and to stop vs code trying to index it.

But technically it's not needed.

Reply:
But neither creating this image or running it write an `mc_data` folder unless you explicitly add it as a volume, right? That makes the scenario pretty specific to that workflow, so I'd rather leave .gitignore clean. Other than that LGTM 🙂

Reply:
Ah yes, I see what you mean. Makes sense :+1: